src/systemctl/systemctl-show.c

```diff
@@ -1989,6 +1990,24 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                         if (r < 0)
                                 return bus_log_parse_error(r);
 
+                        return 1;
+                } else if (streq(name, "ExtraFileDescriptorName")) {
```

Comment:
Should be plural as well

Reply:
Ok! I just used what `OpenFile` did and use singular everywhere. Happy to use plural.

Clarification: When we run systemctl show for a service with 2 extra fds, should we print (1) or (2)

(1)
```
$ systemctl show -p ExtraFileDescriptorNames blah.service
ExtraFileDescriptorNames=test other
```

(2)
```
$ systemctl show -p ExtraFileDescriptorNames blah.service
ExtraFileDescriptorNames=test other
```

(1) is what OpenFile does e.g. each element goes on its own line. This is what the PR currently does

(2) is what I see some other properties do and just concatenate all properties on the same line